assets/styles/pages/homepage.less

```diff
@@ -172,18 +172,24 @@ font-size: @font-size-homepage-normal;
  * "How Shine Works" section
  */
 .how-shine-works {
-  background: black;
+  background-color: rgb(255, 200, 0);
   border-top: 4px solid black;
   padding: 0;
   position: relative;
+  display: flex;
+
+  @media (max-width: @screen-sm-max) {
+    flex-direction: column;
+  }
 
   .product-image {
-    height: 840px;
+    height: auto;
     text-align: center;
+    display: flex;
+    justify-content: center;
     img {
-      max-width: 90%;
-      position: relative;
-      top: @margin-md;
+      max-width: 293px;
```

Comment:
The 293px is kinda tiny in a desktop view. How about keeping this at 90% by default. And then when it drops below @screen-md-max or whichever breakpoint it is when the layout goes single column, then we can set it to 293px or more simply 300px? Is there any reason it's so specific at 293px?